mininet/node.py

```diff
@@ -838,6 +838,38 @@ def init( cls ):
         mountCgroups()
         cls.inited = True
 
+class XIAHost( Host ):
+
+    "XIA host"
+
+    def __init__( self, name, **kwargs ):
+        ""
+       	Node.__init__( self, name, **kwargs )
+	self.xids = {}        
+
+    def config( self, hid=None, **params ):
+	""
+	r = Node.config( self, **params )
+	self.setParam( r, 'setHID', hid=hid )
+	return r
+		
+    def setHID( self, hid=None):
+	""
+	hidList = self.addParameters( hid )	
+	moduleDeps( add='xia_ppal_hid' )
+	for p in hidList:
+	    self.cmd('xip hid new',p)
+	    self.cmd('xip hid getpub',p)
```

Comment:
I don't think this command is necessary, just the `xip hid new` and then `xip hid addaddr`.

Reply:
Yeah you're right, removed that line.